onboard/catkin_ws/src/controls/scripts/drc_utils.py

```diff
@@ -0,0 +1,71 @@
+from tf.transformations import euler_from_quaternion, quaternion_multiply, quaternion_conjugate
+
+
+def get_directions():
+    return ['x', 'y', 'z', 'roll', 'pitch', 'yaw']
+
+
+def get_pose_topic(direction):
```

Comment:
And then also change the name fed in to these places

Reply:
fixed in 189a96e